lib/formatInfo.js

```diff
@@ -49,24 +49,24 @@ function iscustom(schema) {
 function getdefined(schema) {
   if (schema[s.parent]) {
     return {
-      text: `${path.basename(schema[s.filename])}*`,
```

Comment:
What's the background of this change? It seems unrelated.

Reply:
`filename` symbol now holds the basename and `fullpath` symbol holds the full path of the file (not always available)